lldb/source/Plugins/Process/Utility/UnwindDPU.cpp

```diff
@@ -30,13 +30,11 @@ UnwindDPU::UnwindDPU(Thread &thread) : Unwind(thread), m_frames() {}
 void UnwindDPU::DoClear() { m_frames.clear(); }
 
 #define FORMAT_PC(pc) (0x80000000 | ((pc)*8))
-static bool PCIsValid(lldb::addr_t pc) {
-  return pc >= FORMAT_PC(0) && pc < FORMAT_PC(4 * 1024);
-}
 
+#define NB_FRAME_MAX (256)
 bool UnwindDPU::SetFrame(CursorSP *prev_frame, lldb::addr_t cfa,
-                         lldb::addr_t pc) {
-  if (!PCIsValid(pc))
+                         lldb::addr_t pc, Function **fct) {
```

Comment:
the double pointer here is a bit weird, it seems we want to pass it by reference somewhere.
but the GetFunction() API is already this way, so it might be a lost battle.

Reply:
Remove with something cleaner with reference on a ``Address``